static/app/views/alerts/wizard/index.tsx

```diff
@@ -180,19 +180,21 @@ class AlertWizard extends Component<Props, State> {
             <WizardBody>
               <WizardOptions>
                 <Styledh2>{t('Errors')}</Styledh2>
-                {AlertWizardOptions.map(({categoryHeading, options}, i) => (
-                  <OptionsWrapper key={categoryHeading}>
-                    {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
-                    <RadioPanelGroup
-                      choices={options.map(alertType => {
-                        return [alertType, AlertWizardAlertNames[alertType]];
-                      })}
-                      onChange={this.handleChangeAlertOption}
-                      value={alertOption}
-                      label="alert-option"
-                    />
-                  </OptionsWrapper>
-                ))}
+                {getAlertWizardCategories(organization).map(
+                  ({categoryHeading, options}, i) => (
+                    <OptionsWrapper key={categoryHeading}>
+                      {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
```

Comment:
nit pick: 
```suggestion
                      {i > 0 && <StyledH2>{categoryHeading}</StyledH2>}
```

Reply:
If we want to refactor this, I'll change it to `CategoryTitle` so that it's more descriptive. 👍